ReleaseTooling/Sources/ZipBuilder/main.swift

```diff
@@ -105,14 +104,23 @@ struct ZipBuilderTool: ParsableCommand {
           help: ArgumentHelp("The minimum supported iOS version."))
   var minimumIOSVersion: String
 
-  /// The list of architectures to build for.
+  /// The minimum macOS Version to build for.
+  @Option(default: "10.12",
```

Comment:
Nit: This and the one below may be able to fit on one line

Reply:
Done.